circuits/circuits/send.circom

```diff
@@ -33,13 +34,12 @@ template Send(levels, n, k) {
         signal input msghash[k];
         signal input pubkey[2][k];
         // signature verification.
-        VerifySignature(n, k)(
+        VerifySignature(4, n, k)(
             r <== r,
             s <== s,
             msghash <== msghash,
             pubkey <== pubkey,
-            leaf_coins <== leaf_coins,
-            receiver <== recipient,
+            msg <== [leaf_coins[0], leaf_coins[1], highest_coin_to_send, recipient],
```

Comment:
Did `mint.circom` get updated for this change too?

Reply:
mint doesn't have signature, but withdraw is updated to this format too.